packages/pluggableWidgets/feedback-native/e2e/specs/Feedback.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { expectToMatchScreenshot, resetDevice, setText, tapMenuItem } from "../../../../../detox/src/helpers";
+import { element, by, waitFor } from "detox";
+
+describe("Feedback widget", () => {
+    beforeEach(async () => {
+        await tapMenuItem("Feedback");
+    });
+
+    afterEach(async () => {
+        await resetDevice();
+    });
+
+    it("should be able to submit a feedback item", async () => {
+        await expectToMatchScreenshot(element(by.id("feedback1$button")));
```

Comment:
There shouldn't be a difference in sizes, because the local e2e tests should run on the same emulator type.

Maybe, the emulator starts based on whatever previous size it was enlarged/shrunk to? If so, the emulator size should become part of configuration.

Reply:
Fixed. Don't know why it happened.

Reply:
As we discussed, you might had the api version as 32 instead of 30. I've checked the latest screenshots and it works now.